src/webhook_launcher/app/views.py

```diff
@@ -109,159 +127,60 @@ def index(request):
 
             print "Payload to launch:"
             pprint(data, indent=2, width=80, depth=6)
-            launch_queue({"payload" : data})
+            launch_queue({"payload": data})
             print "launched"
 
         except Exception as e:
             print e
-            print "POST with invalid payload from %s" % request.META.get("REMOTE_HOST", None)
+            print "POST with invalid payload from %s" % \
+                request.META.get("REMOTE_HOST", None)
             return HttpResponseBadRequest()
 
         return HttpResponse()
 
     else:
         return HttpResponseNotAllowed(['GET', 'POST'])
 
-class WebHookMappingFilter(filters.FilterSet):
-    package = filters.AllLookupsFilter(name='package')
-    project = filters.AllLookupsFilter(name='project')
-    repourl = filters.AllLookupsFilter(name='repourl')
-    branch = filters.AllLookupsFilter(name='branch')
-    #  user = filters.AllLookupsFilter(name='user__username') ## << FIXME. This is the original line but it seems broken after an update to either django or the filters
-    user = filters.AllLookupsFilter(name='user')
```

Comment:
And another thing is that I dropped the extra dependency to djangorestframework-filters. It is just some extra functionality, which we don't need, on top of the base django_filters.